esi_leap/manager/service.py

```diff
@@ -63,28 +60,29 @@ def stop(self):
 
     def _fulfill_contracts(self):
         LOG.info("Checking for contracts to fulfill")
-        contracts = contract.Contract.get_all_by_status(
-            self._context, statuses.CREATED)
+        contracts = contract.Contract.get_all(
+            self._context, {'status': [statuses.ACTIVE, statuses.CREATED]})
+        now = timeutils.utcnow()
         for c in contracts:
-            if c.start_time and \
-               c.start_time <= timeutils.utcnow():
+            if c.start_time >= now and c.end_time <= now:
```

Comment:
Isn't the opposite of what we want? This seems to be an impossible condition:
```
now -- start time ---------- end time -- now
```
We want:
```
start time -----now------end time
```
Which means start_time <= now and now <= end_time

Reply:
My bad. I did do a test run before but I mustve changed it before making the commit

Reply:
fixed